app/views/browse/routes.html

```diff
@@ -83,7 +83,7 @@
                     </td>
                     <!-- Use shorter Termination title for table-mobile to avoid overlapping text. -->
                     <td data-title="Termination">
-                      {{route.spec.tls.termination}}
+                      {{route.spec.tls.termination | humanizeTLSTermination}}
```

Comment:
Can you humanize this as well on the service page (which shows routes in a table) and the route details page?